packages/create-cloudflare/e2e-tests/workers.test.ts

```diff
@@ -137,9 +188,6 @@ describe
 							// Relevant project files should have been created
 							expect(project.path).toExist();
 
-							const gitignorePath = join(project.path, ".gitignore");
-							expect(gitignorePath).toExist();
-
```

Comment:
The asset-only template has no `.gitignore`. I could have add an options to skip this only for that template but I think it is unnecessary to assume `.gitignore` exists on all template especially when the git setup on c3 is optional.